plumbum/cli/switches.py

```diff
@@ -464,18 +464,47 @@ class MyApp(Application):
                              comparison or not. The default is ``False``
     :param csv: splits the input as a comma-separated-value before validating and returning
                 a list. Accepts ``True``, ``False``, or a string for the separator
-    """
+    :param all_markers: when a user inputs any value from this set, it is considered that
+                        he requested iteration over all values. By default `all_markers`
+                        are {"*", "all"}."""
```

Comment:
```suggestion
                        are {"*"}; you can set it to {"all", ","}, for example."""
```

Reply:
Why do you suggest it be changed, if we resolve conflicts automatically?

Reply:
Plumbum comes with built-in translations for German, French, Dutch, and Russian. Either this string needs to support translations, or it should be language agnostic. A local dependent default is poor (as you noted), so why not just use the symbol? Also, if the user uses all-caps for their options, then this will be lower case, which is weird. Why not just use `"*"` as the default string (exactly the same number of characters as `all`) and let the user decide if they like "all" or something else?

Also there's the issue of being misleading. "all" is much more likely to look like a normal option that "*" is; and it's not a normal option, it just selects all the others.

I don't think anyone else has an "all/*" like option. Does anyone else? I checked Click, doesn't seem to have this sort of select-all feature.

Reply:
Does this show up in the help anywhere?

If you really want to make "all" part of the default, I'd take a tie breaker opinion. @tomerfiliba perhaps? Or any other person who has committed anything.